packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1554,6 +1581,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     ) {
         this.addMetadataToSummary(summaryTree);
 
+        if (this.runtimeOptions.enableRuntimeIdCompressor && this.idCompressor !== undefined) {
+            const idCompressorState = JSON.stringify(this.idCompressor.serialize(false));
```

Comment:
Do we always want to serialize with no session? I think we probably do; just double checking. I noticed you handle both cases when deserializing above, but that's probably just because it's not hard to do so you may as well? I.e. we don't actually expect any serialized ones with sessions

Reply:
I'm actually not sure what the answer is to this question. Looking through experimental SharedTree, I only found a couple instances where we call `serialize` with this flag set to true and they're in test files. From the doc comment it looks like it includes unfinalized state in the serialized compressor when it's set to true, but I'm not sure how/where that would be used practically (stashed ops?). Do you or @taylorsw04 remember?

Reply:
It is used for the "long offline scenarios", when a client is disconnected from the network for a long time (e.g. days) but wants to work on a document during that time. They might close and reopen the app many times, and the flag here allows that to be considered "the same session" with respect to IDs (e.g. they keep the same session ID).

I don't know if we eventually want to support that at the container/runtime level or not. Probably, someday? And almost centainly not right now, so, this code seems good; I was just making sure that everything here was intentional.